### ISSUE TITLE
Create integration tests

### ISSUE THREAD

**Issue by forgedhallpass** [OWASP/www-project-csrfguard#192]
Currently the integration of the solution is manually validated through the test application.

It would be much nicer to automate the scenarios and make it part of the CI build.

**Comment by ecabrerar** [comment]
I'm excited to learn how I can contribute to this issue. I need more details about the scenarios that you want to automate @forgedhallpass

**Comment by ecabrerar** [comment]
Hi @forgedhallpass , please assign the ticket to me.

**Comment by forgedhallpass** [comment]
@ecabrerar assigned

**Comment by ecabrerar** [comment]
Yes, I'm starting to work on it. I don't have much experience with selenium, but I'm learning in the process, which is why I'm moving slowly.

**Comment by ecabrerar** [comment]
Yes, I'm starting to work on it. I don't have much experience with selenium, but I'm learning in the process, so I'm moving slowly.

**Comment by forgedhallpass** [comment]
@ecabrerar any updates?
